hub/hub.html

```diff
@@ -40,7 +40,7 @@ <h3 class="research-hub-sub-title">Explore and extend models<br> from the latest
 
           <hr class="hub-divider">
 
-          <div id="hub-index-cards">
+          <div id="hub-cards">
```

Comment:
After you look into the `listId` issue in "filter-hub-tags.js", I think you can change this back to "hub-index-cards".